packages/Laravel/tests/Rector/Class_/InlineValidationRulesToArrayDefinitionRector/InlineValidationRulesToArrayDefinitionRectorTest.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Laravel\Tests\Rector\Class_\InlineValidationRulesToArrayDefinitionRector;
+
+use Rector\Laravel\Rector\Class_\InlineValidationRulesToArrayDefinitionRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class InlineValidationRulesToArrayDefinitionRectorTest extends AbstractRectorTestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        parent::setUpBeforeClass();
+
+        eval('
```

Comment:
Instead of this better add a stub, see `/Source` of other tests

Reply:
This is made to create specific namespace (`Illuminate\Foundation\Http\Request`) without installing dependency. But if I create a source file with this namespace, it will be proposed to import in projects that include rector. That's why I decided to create this class in runtime

Reply:
See `stubs` for it